src/transaction/log_manager.c

```diff
@@ -14350,6 +14351,8 @@ cdc_get_lsa_with_start_point (THREAD_ENTRY * thread_p, time_t * time, LOG_LSA *
   while (!LSA_ISNULL (&process_lsa))
     {
       log_rec_header = LOG_GET_LOG_RECORD_HEADER (log_page_p, &process_lsa);
+
+      LSA_COPY (&cur_log_rec_lsa, &process_lsa);
```

Comment:
1. OK, it seems good
2. When `process_lsa` is need to be returned, process_lsa does not point to the starting point of the log record header. This is because process_lsa moves forward as it reads the log record header and reads the data part of the log. (`LOG_READ_ADD_ALIGN`)

Reply:
oh, you're right. I missed that. Thank you.